full-service/src/config.rs

```diff
@@ -87,21 +87,6 @@ pub struct APIConfig {
     #[structopt(flatten)]
     pub peers_config: PeersConfig,
 
-    /// Quorum set for ledger syncing. By default, the quorum set would include
```

Comment:
This stuff moved into `PeersConfig` with the idea that now `PeersConfig` contains all the information needed on how to connect to a set of consensus nodes.